docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -1129,6 +1129,36 @@ If you were using them, you need to replace them with the following code:
     +  extends BaseMultiInputPickersTextFieldProps<true> {}
     ```
 
+## ✅ Rename `date-fns` adapter imports
+
+:::warning
```

Comment:
Added a warning here to align with the codemod package readme.
Does it sound like a decent compromise @flaviendelangle @michelengelen? 🤔 
We can always remove it from the `preset-safe` and keep it opt-in if there are complaints... 🤷